pkg/apis/operator/v1alpha1/tektonpipeline_types.go

```diff
@@ -161,6 +163,8 @@ type PipelinePerformanceProperties struct {
 	PipelinePerformanceLeaderElectionConfig `json:",inline"`
 	// +optional
 	PipelineDeploymentPerformanceArgs `json:",inline"`
+	// +optional
+	Replicas *int32 `json:"replicas,omitempty"`
```

Comment:
If different values are set to `replicas` by Options as well as spec  then which one will be used ?

Reply:
always `options` value will be final